### DISCUSSION TITLE
[ FF7 ] Crash during Nibelheim flashback

### DISCUSSION THREAD

**Discussion by RaulDiago** [julianxhokaxhiu/FFNx#349] [Q&A] [answered]
The game always crashes right before the dragon fight starts in the Nibelheim flashback. I've tried with and without mods, with and without reunion, 1998 version and Steam, it always breaks at the same point.
I've read about replacing the audio.dat and audio.fmt files, but I can't find them anywhere.

The error occurs both on my laptop with integrated intel graphics and on my desktop PC with GTX 960 with updated drivers, both with windows 10. I have tried compatibility mode with winows XP, 98 and 95, it does not work.

Here is the FFNx.log.
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/7875412/FFNx.log)

**Comment by julianxhokaxhiu** [comment, marked as answer]
As you're using 7th all I can do is suggesting you to ask for support on their relative channels. FFNx support is provided only when installing it vanilla. Cheers!

**Reply by RaulDiago** [reply]
No, i'm not using 7th, as I said in the previous message. I've installed it vanilla and I have the same problem.

**Reply by julianxhokaxhiu** [reply]
In the loaded DLL list of your log, there is `EasyHook.dll` which is used by 7th only. Are you sure you're not using it?

Also, you're using a very old release of FFNx. Please update to the latest stable.

**Reply by RaulDiago** [reply]
I used 7th. Having the error I wanted to pass this part with the vanilla version and I get the same error. Maybe that's why it shows up.
I have updated and continue with the same error. I attach again the FFNx and a video

https://user-images.githubusercontent.com/23319306/149675227-d611a3d6-0437-4c85-8d95-79c5a2ec25cb.mp4

[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/7877780/FFNx.log)
.

**Reply by julianxhokaxhiu** [reply]
That crash is nothing related to FFNx. The error message you get is from Reunion I suppose. Also the version is still at 1.9.0.0 which means you're not using the latest stable. But I'm 100% sure this is not a vanilla FFNx install, therefore I can't help you sorry.

**Reply by RaulDiago** [reply]
I have found the error using the black chocobo saves editor. For some reason, sepfiroth has been changed to vincent in the party, and that crashes the game (sephiroth vs dragon).
Anyway, thank you very much for your time.

**Reply by julianxhokaxhiu** [reply]
Glad to know you've found the core issue. Cheers.
